eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -47,6 +47,7 @@ steps:
 
   - pwsh: |
         $(Build.SourcesDirectory)/scripts/Slngen.ps1 -All -NoLaunch
+        $(Build.SourcesDirectory)/.dotnet/dotnet sln add --project $(Build.SourcesDirectory)/src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj
```

Comment:
@carlossanlop did you try this command locally?

Reply:
```suggestion
        $(Build.SourcesDirectory)/.dotnet/dotnet sln add $(Build.SourcesDirectory)/src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj
```